internal/lib/dynamic_client.go

```diff
@@ -313,13 +313,13 @@ func GetCNIPlugin() string {
 // is complicated business right now.
 func WaitForInitSecretRecreateAndReboot() {
 	cs := utils.GetInformers().ClientSet
-	if err := cs.CoreV1().Secrets("vmware-system-ako").Delete(context.TODO(), "avi-init-secret", metav1.DeleteOptions{}); err != nil {
+	if err := cs.CoreV1().Secrets("vmware-system-ako").Delete(context.TODO(), AviInitSecret, metav1.DeleteOptions{}); err != nil {
```

Comment:
dont you need a nil check here ?

Reply:
I did not understand, nil check for what? I don't think we need any nil check here